src/scripts/sssom_to_robot_template.py

```diff
@@ -0,0 +1,52 @@
+"""Convert SSSOM to ROBOT template."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+from sssom import MappingSetDataFrame
+from sssom.parsers import parse_sssom_table
+
+ROBOT_ROW = {
+    'subject_id': 'ID',
+    'subject_label': '',
+    'object_id': 'A oboInOwl:hasDbXref',
+    'object_label': '',
+    'equivalence': '>A oboInOwl:source',
+    'ORCID': '>A oboInOwl:source SPLIT=|',
+}
+
+
+def sssom_to_robot_template(inpath: Union[str, Path], outpath: Union[str, Path]):
+    """Convert SSSOM to ROBOT template."""
+    msdf: MappingSetDataFrame = parse_sssom_table(inpath)
+    df: pd.DataFrame = msdf.df
+
+    # Filter
+    df = df[df['predicate_id'] == 'skos:exactMatch']
+
+    # Conversion
+    df = df[['subject_id', 'subject_label', 'object_id', 'object_label']].sort_values(['subject_id', 'object_id'])
+    df['equivalence'] = 'MONDO:equivalentTo'
+    df['ORCID'] = '|'.join(msdf.metadata['creator_id'])
+    df = pd.concat([pd.DataFrame([ROBOT_ROW]), df])
```

Comment:
add another column:

df['mapping_provider'] = msdf.metadata['mapping_provider'] and have it be added the as `A oboInOwl:hasDbXref` just as the others.

Reply:
When you say add it to the `'object_id': 'A oboInOwl:hasDbXref'`, you mean add it as `'mapping_provider': '>A oboInOwl:source'`? This is what I've done. Can you confirm that's correct?

Reply:
yes!